src/main/java/nextstep/ladder/entity/ladder/RandomLinkGenerator.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder.entity.ladder;
+
+import java.util.Random;
+
+public class RandomLinkGenerator implements LinkGenerator{
+
+    private static final RandomLinkGenerator instance = new RandomLinkGenerator();
+    private static final Random random = new Random();
+    private static final int BOUND = 2_000_000_000;
+
+    private RandomLinkGenerator() {}
+
+    public static RandomLinkGenerator getInstance() {
+        return instance;
+    }
+
+    @Override
+    public boolean isPossibleToLink() {
+        return random.nextInt(BOUND) % 2 == 0;
```

Comment:
다음과 같이 변경해도 좋을 것 같아요 :)
```suggestion
        return random.nextBoolean();
```